### ISSUE TITLE
docs: update Bundly readme

### ISSUE THREAD

**Issue by sauravhiremath** [bundly/dash-beta#1]
- [ ] Updated logo
- [ ] Remove unnecessary information
- [ ] Remove outdated gif links

**Comment by kartikcho** [comment]
Also remove information that limit the project's features to the Fellowship, let's write a more general purpose doc.

**Comment by darshkpatel** [comment]
I think we should do that after we make it more general purpose

**Comment by kartikcho** [comment]
I agree but the recent additions are aiming to make it more general purpose soon, the docs should reflect that already and all the features we'll be adding. That way people will know what features will be added/ are in the app already. 

Besides a one time rewrite that would be relevant in the long run seems better than rewriting the doc again in the future imo.
